app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -1109,7 +1100,7 @@ public void onClick(View view) {
      */
     private void showSoftKeyboard() {
         InputMethodManager imm = (InputMethodManager) getSystemService(Context.INPUT_METHOD_SERVICE);
-        imm.toggleSoftInput(InputMethodManager.SHOW_IMPLICIT,InputMethodManager.HIDE_IMPLICIT_ONLY);
+        imm.toggleSoftInput(InputMethodManager.SHOW_IMPLICIT, InputMethodManager.HIDE_IMPLICIT_ONLY);
```

Comment:
This is because of formatting of the code.Should I include it in a different commit??